### DISCUSSION TITLE
how to push from cli

### DISCUSSION THREAD

**Discussion by flog89** [community/community#135204] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

how to push a repo from cli

**Comment by mu6m** [comment]
there are some instructions when you create your repo copy and paste them in your cli make sure that you have the ssh keys setup

**Reply by flog89** [reply]
thanks

**Comment by ebndev** [comment]
Hi @flog89 & @mu6m, thanks for being a part of the GitHub Community!

While we appreciate you wanting to contribute to Discussions, please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
